ihmc-graphics/build.gradle.kts

```diff
@@ -45,7 +45,8 @@ jmonkeyengineTestDependencies {
 }
 
 libgdxDependencies {
-   api(ihmc.sourceSetProject("main"))
+   api("us.ihmc:ihmc-java-toolkit:source")
+   api("us.ihmc:ihmc-robotics-toolkit:source")
```

Comment:
This cuts out:
ihmc sensor processing
ihmc communication
ihmc interfaces
ihmc robot models
vecmath
ihmc robot description
and this stuff
![image](https://github.com/user-attachments/assets/8813bbda-ca9b-43fa-b372-c0e5b51a70fd)
![image](https://github.com/user-attachments/assets/d42cd45b-9256-4401-94b1-509c9af7aa5f)